src/leiningen/new/chestnut/src/clj/chestnut/server.clj

```diff
@@ -5,7 +5,13 @@
             [ring.middleware.defaults :refer [wrap-defaults {{ring-defaults}}]]
             [ring.middleware.gzip :refer [wrap-gzip]]
             [ring.middleware.logger :refer [wrap-with-logger]]
-            [environ.core :refer [env]]{{{server-clj-requires}}})
+            [environ.core :refer [env]]
+            [com.stuartsierra.component :as component]
+            [reloaded.repl]
+            (system.components
```

Comment:
I'm not a huge fan of these nested requires (anymore). I would prefer to have one vector/line per required namespace.